src/test/java/com/example/demo/RestTest.java

```diff
@@ -52,4 +53,71 @@ public void testAuthentication() throws Exception {
                 .andDo(print())
                 .andExpect(status().isOk());
     }
+
+    @Test
+    public void testSurvey() throws Exception {
+        // Create user
+        this.mockMvc.perform(post("/user")
```

Comment:
Does this test requires logging in for the user?

Reply:
Yes